distributedID/blockchain-explorer
# Adding the explorer stop logic

### Ref: 
#48 

### Description:
`npm` takes care of stopping of the `explorer` when the `Ctrl + C` is executed.

### Blocking:
No

### Documentation:
Nope

### Testing:

1. Use `npm start` to start the `explorer`.
1. Use `Ctrl + C` to stop the `explorer`.
1. Use `npm start` to start the `explorer` again. 

In the second attempt, there should be no error.
